OpenRA.Mods.Common/Traits/Health.cs

```diff
@@ -23,6 +23,7 @@ public class HealthInfo : ITraitInfo, UsesInit<HealthInit>
 		public readonly bool NotifyAppliedDamage = true;
 
 		[FieldLoader.LoadUsing("LoadShape")]
+		[Desc("Capsule, Circle and Rectangle are supported.")]
```

Comment:
`By default, Capsule...` then imo. This is still useful for modders to know what to look up in the yamls.